website/src/components/astGrep/template/source.ts

```
@@ -0,0 +1,16 @@
+// console.log() will be matched by pattern!
+// click diff tab to see rewrite.
+
+function tryAstGrep() {
+  console.log('matched in metavar!')
+}
+
+const multiLineExpression =
+  console
+   .log('Also matched!')
+
+if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)